src/site/twitch.tv/modules/chat/components/message/UserMessage.vue

```diff
@@ -153,6 +153,7 @@ const timestamp = intlFormat(
 		localeMatcher: "lookup",
 		hour: "numeric",
 		minute: "numeric",
+		second: "numeric",
```

Comment:
This would make it permanently show seconds, which is not a default behavior. It must be togglable, and off by default

Reply:
like adding a new option in the 7tv settings at Chat > Style ?

Reply:
@TheRealGI I expect it makes the most sense to put it there. I'm looking forward to having seconds. 😄